util/string_util.cc

```diff
@@ -437,6 +437,45 @@ bool SerializeIntVector(const std::vector<int>& vec, std::string* value) {
   return true;
 }
 
+int ParseTimeStringToSeconds(const std::string& value) {
+  int hours, minutes;
+  char colon;
+
+  std::istringstream stream(value);
+  stream >> hours >> colon >> minutes;
```

Comment:
Not a big deal, but cases like "0:1-2:000000000000000042" are considered valid. It doesn't bother me but worth noting in the implementation and/or test if not fixing.

Reply:
That's a weird thing for users to try but sure, let me add it to the test. As long as it's a valid number format that we can parse, there's no reason to block it I think.